@navikt/ds-react/stepper/src/Stepper.tsx

```diff
@@ -0,0 +1,67 @@
+import "@navikt/ds-css/stepper/index.css";
+import cl from "classnames";
+import React, { createContext, forwardRef } from "react";
+import { uuid } from "../../util/src";
```

Comment:
Jeg ser at util er definert utenfor denne komponenten. Må den ikke da defineres som en avhengighet? Jeg tror ikke jeg forstår `yarn workspaces` helt.

Reply:
Oppsettet vi har nå for `ds-react` bryr seg bare om hva `ds-react/package.json` sier. Package.json som ligger under hver komponent brukes egentlig bare for å holde styr på hvilken dependencies hver komponent bruker, men har ingen nødvendig funksjon ellers. 
Dette vil da si at alle dependencies for hver komponent også må ligge i `ds-react/package.json` for at alt skal fungere riktig.